package Protos;

message MatchingPursuitConfig {
    required int32 seed = 1;
    required double threshold = 2;
    required uint32 learn_iterations = 3;
    required uint32 jobs = 4;
    required double learning_rate = 5;
    required uint32 filters_num = 6;
    required uint32 filter_size = 7;
    required bool learn = 8;
    required bool continue_learning = 9;
    required uint32 batch_size = 10;
    required double noise_sd = 11;
}

message FilterMatch {
    required uint32 fi = 1;
    required double s = 2;
    required uint32 t = 3;
}

